src/sklearn_knn/original/_mahalanobis_transformer.py

```diff
@@ -0,0 +1,18 @@
+import numpy as np
+from sklearn.base import BaseEstimator, TransformerMixin
+
+from .._base import MyStandardScaler
+
+
+class MahalanobisTransformer(TransformerMixin, BaseEstimator):
+    def fit(self, X, y=None):
+        self.scalar_ = MyStandardScaler().fit(X)
```

Comment:
Does it make more sense to call this a `scaler_` than a `scalar_`? Scalar makes me think of the mathematical or programming meaning, which I don't think apply here?

Reply:
Yep, of course you're right